src/components/features/Aircraft.js

```diff
@@ -3,7 +3,7 @@ import aircraftData from "../initialData/aircraftData";
 
 function Aircraft({ onSelectAircraft }) {
   return (
-    <select onChange={(e) => onSelectAircraft(e.target.value)}>
+    <select onChange={(plane) => onSelectAircraft(plane.target.value)}>
```

Comment:
use event please, even if it seams nitpicky. We want you to write the best code you can, and in this situation the goal is communicate clearly to other developers what is going on.

Reply:
Many thanks for the hints, Changes made.